docs/data/Allas/using_allas/swift_client.md

```diff
@@ -0,0 +1,380 @@
+```bash
+RANDOMKEY="my-super-secret-key"
+```
+Post a Temp-URL-Key to the whole project. **Please note:** If someone changes this project wide Temp Key, all Temp URLs stop working. You should coordinate changes like these within your computing project.
+```bash
+swift post -m "Temp-URL-Key:$RANDOMKEY" 
+```
+To get your <i>OS_PROJECT_ID</i> use command `env`:
+```bash
+$ env | grep -i project
+OS_PROJECT_NAME=project_123456
+OS_PROJECT_ID=<os_project_id>
+```
+Save the full path to the Swift object (Replace the part *"os_project_id"* with your OS_PROJECT_ID):
+
+```bash
+MYURL=https://object.pouta.csc.fi/swift/v1/AUTH_"os_project_id"/my_fishbucket/bigfish.jpg
+```
+
+
+Create a Temp-URL-Key valid for 86400 seconds (24 hours):
+```bash
+$ swift tempurl GET 86400 $MYURL $RANDOMKEY
+https://object.pouta.csc.fi/swift/v1/AUTH_6e3f5db8e08940f481744240af8701e5/my_fishbucket/bigfish.jpg?temp_url_sig=9a118ddda22c83c7a6cd49c013389f0507c007ca&temp_url_expires=1514648675
+```
+
+Use the previously created Temp URL to download the object:
+```bash
+$ curl https://object.pouta.csc.fi/swift/v1/AUTH_6e3f5db8e08940f481744240af8701e5/my_fishbucket/bigfish.jpg?temp_url_sig=9a118ddda22c83c7a6cd49c013389f0507c007ca&temp_url_expires=1514648675> bigfish.jpg
+```
+You may set a second key by adding another metadata entry with title "*Temp-URL-Key-2*".
+
+&nbsp;
+
+## Giving another project read and write access to a bucket
+
+You can give another project read and write rights to a bucket. Giving project _project1_ read right to bucket <i>my_fishbucket</i> is done with command:
+```bash
+swift post my_fishbucket -r "project1:*"
+```
+Write access can be given similarly by replacing the _-r_ (_read_) with _-w_ (_write_):
+```bash
+swift post my_fishbucket -w "project1:*"
+```
+The sign _*_ after the project name defines that all the project members in the project gets the rights.
+
+Alternatively, you can give read and write access only to certain members of another project:
+```bash
+swift post my_fishbucket -r "project2:member1"
+swift post my_fishbucket -w \
+   "project3:member1,project3:member2,project5:member1,project6:*"
+```
+
+**Please note:** If you have allowed access for specific projects, making the shared project public and private again will remove the previous access permissions.
+
+In case you allow _-w_ access for another project, the members of the other project can upload files to your bucket and remove your files. 
+However, you have not access to those uploaded files until either you or the sender shares the bucket with your project:
+```bash
+swift post <your_bucket_name> -r "your_project:*"
+```
+For example:
+```bash
+swift post my_fishbucket -r "project_1234:*,project_4567:*"
+```
+Alternatively, you can set the project public and then access the file.
+
+
+&nbsp;
+
+
+
+## Files larger than 5 GB
+
+Since the OpenStack command does not support uploading files larger than 5GB, we can use Swift to upload a so-called _Static Large Object_ (SLO).
```

Comment:
This is missleading. It does support uploading larger files and this section explains how to do it(!) Should be rephrased along the quote ne paragraph down